dist/dropdown/ds6/dropdown.css

```diff
@@ -190,9 +190,14 @@ span.fake-menu__status {
   box-sizing: border-box;
   width: 100%;
 }
-.listbox__control > input {
+.listbox__control > input,
+.listbox.listbox--mobile > select.listbox__native {
```

Comment:
I'd rather not call this `mobile`, as it can be used on large screens too.

Reply:
Right. I struggled with what to call this version.

Reply:
`native`?

Reply:
For 4.0.0, we could convert this new, simple version into it's own `select` module. Then listbox gets renamed too `combobox` module.

Reply:
If we split this into two components, it might be easier altogether. Hmmm.

Reply:
How would we help developers distinguish between the two? Would we let design dictate that?

Reply:
Maybe: `<ebay-select type="native"`>, `<ebay-select type="custom">`

For pure CSS, without coreui, combobox would have two flavors: writeable (i.e. an autocomplete/typeahead) and readonly (i.e. a custom select).